### ISSUE TITLE
t value of co-integration test is 0 for two identical arrays

### ISSUE THREAD

**Issue by XuewanZhao** [statsmodels/statsmodels#4596]
Hi,
As I'm trying to use `ts.coint` function in `tsa.stattools`, there is a weird problem.

```In [29]: a = range(0,5)

In [30]: b = range(0,5)

In [31]: ts.coint(a,b)
D:\ANACONDA\lib\site-packages\statsmodels\tsa\stattools.py:1018: UserWarning: y0 and y1 are perfectly colinear.  Cointegration test is not reliable in this case.
  warnings.warn("y0 and y1 are perfectly colinear.  Cointegration test "
Out[31]: (0, 0.9859002580259643, array([-8.7298525, -5.2900925, -4.27475  ]))```

You can see that as a and b are identical, the t value for these two arrays should be very large in absolute value and the p value should be 0. However, the output of t value is 0 while p value is almost 1. Is this a bug?

**Comment by josef-pkt** [comment]
see #4237
I fixed this for the 0.9 release

**Comment by XuewanZhao** [comment]
@josef-pkt Hi Josef,
As I trying to update statsmodels to 0.9 release, I met many problems.
1. If I try to update this package with Anaconda, it can only give me 0.8 version.
2. I tried download the source zip file and put it under site-packages, and then I tied the methods in the file: INSTALL.txt
 2.1 `pip install statsmodels --upgrade-strategy only-if-needed` can only find 0.8 version.
 2.2 Then I tried `python setup.py install`, however, new folders `statsmodels` and `statsmodels.egg-info` are generated under the folder I extracted from the zip file. I can't do `import statsmodels.tsa` now. 
    2.2.1 I tried to move these two folders out and under the folder `site-packages`, however, one error raised:`ImportError: Building module statsmodels.tsa.kalmanf.kalman_loglike failed: ["distutils.errors.CompileError: command 'D:\\\\Visual Studio\\\\VC\\\\BIN\\\\x86_amd64\\\\cl.exe' failed with exit status 2\n"]`
   2.2.2 Then I deleted the two folder I moved out and I tried use `import statsmodels.statsmodels.tsa', however, the new error raised `ModuleNotFoundError: No module named 'statsmodels.tsa.arima_model'`. I guess it's due to the outside folder`statsmodels` outside `statsmodels`.

I'm really confused now... I don't know how can I use 0.9 version lol. Any suggestion?

**Comment by josef-pkt** [comment]
you can use the binary wheels for the release candidate in https://pypi.org/project/statsmodels/0.9.0rc1/#files for the `pip install`
those wheels were build by appveyor

(I haven't managed yet to build binaries myself for python > 3.4 on Windows.)

**Comment by XuewanZhao** [comment]
@josef-pkt Hi Josef, 
I installed the wheel file and restarted spyder try to see if coint function is good now. However, there are two problems occured:
1. Spyder still calls 0.8 version statsmodels
2. I replaced the tsa folder in 0.8 version with tsa folder from 0.9 version, and then I tried to import it. In the file `tsa\__init__.py`, there is `from statsmodels import PytestTester`. However, there is no such a folder or file no matter in 0.8 version nor 0.9 version. I checked the code on git and there is no such a folder either....that's really weird. How can I solve this?
(Yes, my python is 3.6 version. ;) )

**Comment by MeghashreeRaghav** [comment]
Hi XuewanZhao 

can you please explain on the resolution you found for this issue.

Even i am facing the similar issue.

Thanks,
Megha

**Comment by XuewanZhao** [comment]
Hi  Meghashree,

You can use the binary wheels for the release candidate in
https://pypi.org/project/statsmodels/0.9.0rc1/#files for the pip install
those wheels were build by appveyor.

You can also see #4237
<https://github.com/statsmodels/statsmodels/issues/4237>.
The author fixed this for the 0.9 release.

Best,
Xuewan
